nvdaHelper/espeak/sconscript

```diff
@@ -135,6 +136,18 @@ def espeak_compilePhonemeData_buildAction(target,source,env):
 	espeak.espeak_ng_CompilePhonemeData(22050,None,None)
 	espeak.espeak_Terminate()
 
+
+# refer to CLEANFILES in include\espeak\Makefile.am 
+for f in (
+		os.path.join(espeakRepo.abspath, "dictsource", "ru_listx"),
+		os.path.join(espeakRepo.abspath, "dictsource", "cmn_listx"),
+		os.path.join(espeakRepo.abspath, "dictsource", "yue_listx"),
+	):
+	if os.path.exists(f):
+		print(f"Removing listx file: {f}")
+		os.remove(f)
+
+

```

Comment:
Are these files generated during the build?
This code means the files are removed as a side-effect of loading the sconscripts, rather than during a build or clean action. IE if the developer intent is no action, this would still delete the files, right?

Reply:
I believe these are files that need to be cleaned before the build:
https://github.com/espeak-ng/espeak-ng/blob/master/.gitignore#L194-L196.

They are on the projects gitignore, so they shouldn't be a requirement for the build.

Reply:
Cleaning the CLEANFILES and emoji files can be turned into a `PreAction` to be safer.